api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -191,4 +191,13 @@ public void validate_shouldRejectNumericReactionValue() {
 		validator.validate(allergy, errors);
 		assertTrue(errors.hasErrors());
 	}
+	
+	@Test
+	public void validate_shouldRejectNumericAllergenValue() {
```

Comment:
Can you also provide proof the Validator message is returned ?

Reply:
Yes @tendomart just still with a blocker SDK not running throws error

Reply:
Hi @tendomart  please what is your progress on this PR. I see it closed are you still to come back ?